packages/grpc-js/src/channel.ts

```diff
@@ -230,6 +239,14 @@ export class ChannelImplementation implements Channel {
       new MaxMessageSizeFilterFactory(this.options),
       new CompressionFilterFactory(this),
     ]);
+
+    this.callRefTimer = setInterval(() => {}, 1 << 31 - 1);
```

Comment:
Also, https://www.npmjs.com/package/node-noop :-P

Reply:
Sadly, that's not even the only package that provides a noop function. We used to depend on one of them.